plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -625,8 +625,8 @@ public void testQueryCache()
             assertThatThrownBy(() -> query(createNeverDisposition, "SELECT * FROM test." + materializedView))
                     .hasMessageContaining("Not found");
             // Populate cache and verify it
-            assertQuery(queryResultsCacheSession, "SELECT * FROM test." + materializedView, "VALUES 5");
-            assertQuery(createNeverDisposition, "SELECT * FROM test." + materializedView, "VALUES 5");
+            assertEventually(() -> assertQuery(queryResultsCacheSession, "SELECT * FROM test." + materializedView, "VALUES 5"));
+            assertEventually(() -> assertQuery(createNeverDisposition, "SELECT * FROM test." + materializedView, "VALUES 5"));
```

Comment:
Can we revert this change? I think you changed these lines to fix flakiness which is described in the commit body. I don't see the failure of this test in master branch.